cmd/nfs_threads.py

```diff
@@ -33,18 +35,26 @@
 H2 = ['arrived', 'enqueued', 'woken', 'used', 'calls', 'iops', 'thruput',
       'iops', 'thruput']
 
-INTERVAL = 5
+
+def parse_cmdline():
+    parser = argparse.ArgumentParser(
+        description='Display nfsd thread usage info along with NFS I/O '
+        'context')
+    parser.add_argument(
+        '--interval', nargs='?', type=int, choices=range(1, 31),
```

Comment:
What does `nargs='?'` do here?

Reply:
good catch.  Not sure where I picked that up.  In this context it allows using `--interval` without a value which is not expected or desired.  I'll removed it.